### PR TITLE
feat:RDCC-1589 Assign access a case- add organisationIdenitifier in get user response

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-1589

### Change description ###
feat:RDCC-1589 Assign access- add organisationIdenitifier in get user response

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java [hunk 1]
```diff
@@ -233,69 +233,41 @@ public void findUsersByOrganisation_with_status_active() throws Exception {
 
         ResponseEntity responseEntity = professionalUserService.findProfessionalUsersByOrganisation(organisation, "false", true, "Active");
 
+        ProfessionalUsersEntityResponse professionalUsersEntityResponse1 = ((ProfessionalUsersEntityResponse)responseEntity.getBody());
+
         Mockito.verify(professionalUserRepository, Mockito.times(1)).findByOrganisation(organisation);
```
Comment (KOTLAPRA23) [inline src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java:238]: could you please move Mockito to package level

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java [hunk 2]
```diff
@@ -233,69 +233,41 @@ public void findUsersByOrganisation_with_status_active() throws Exception {
...
+    public void test_findUsersByOrganisation_failure_in_Uo() throws Exception {
+        HttpHeaders header = new HttpHeaders();
+        header.setContentType(APPLICATION_JSON);
+        List<ProfessionalUser> users = new ArrayList<>();
+        users.add(professionalUser);
+        when(professionalUserRepository.findByOrganisation(organisation)).thenReturn(users);
+        ErrorResponse errorResponse = new ErrorResponse("some error message", "some error description", "21:10");
+        when(userProfileFeignClient.getUserProfiles(any(), any(), any())).thenReturn(Response.builder().request(mock(Request.class)).body(mapper.writeValueAsString(errorResponse), Charset.defaultCharset()).status(400).build());
+        ResponseEntity responseEntity = professionalUserService.findProfessionalUsersByOrganisation(organisation, "false", true, "");
+        Mockito.verify(professionalUserRepository, Mockito.times(1)).findByOrganisation(organisation);
```
Comment (KOTLAPRA23) [inline src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java:262]: move Mockito to package level

File: src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java [hunk 3]
```diff
@@ -1,236 +1,239 @@
...
+    }
+
+    private Page<ProfessionalUser> getPagedListOfUsers(Organisation organisation, Pageable pageable) {
+        Page<ProfessionalUser> professionalUsers = professionalUserRepository.findByOrganisation(organisation, pageable);
+
+        if (professionalUsers.getContent().isEmpty()) {
+            throw new ResourceNotFoundException("No Users found for page number " + pageable.getPageNumber());
+        }
+
+        return professionalUsers;
```
Comment (KOTLAPRA23) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java:173]: why entire class is showing difference here?

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java (hunk 1 above)
Comment (shreedhar-hmcts) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java]: done

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java (hunk 2 above)
Comment (shreedhar-hmcts) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java]: done

**Comment by shreedhar-hmcts** [comment]
Closing this as one file has CRLF Vs LF changes and difference is not shown in git.

**Pull request closed without merging**